src/static/js/modules/form-validation.js

```diff
@@ -50,14 +50,38 @@ function _clearNotification( elem ) {
   $elem.find( '.error' ).removeClass( 'error' );
 }
 
+function _sendSubscriptionRequest( elem ) {
+  event.preventDefault();
+  var $form = $( elem );
+  var action = $form.attr( 'action' );
+  var formData = $form.serialize();
+
+  $.post( action, formData, function( data ) {
```

Comment:
Since there are no closures in the callback function consider moving it to its own top-level function.